src/plugins/kibana/public/discover/index.html

```diff
@@ -1,5 +1,53 @@
 <div ng-controller="discover" class="app-container">
-  <navbar name="discover">
+  <navbar name="discover-options">
+    <div class="discover-info">
```

Comment:
This should be generalized into a style that can be shared across visualize/dashboard/discover since they all use a title style